Talkish.API/Controllers/BlogsController.cs

```diff
@@ -47,21 +51,22 @@ public async Task<IActionResult> GetBlogTopicsByBlogId([FromRoute] int Id)
             return Ok(topicDTOs);
         }
 
+        [Route("", Name = "createBlogRoute")]
         [HttpPost]
         public async Task<IActionResult> CreateBlog([FromBody] AddBlogDTO BlogData)
         {
             Blog blog = _mapper.Map<Blog>(BlogData);
             await _service.CreateBlog(blog);
-            return Ok(BlogData);
+            return Created("createBlogRoute", BlogData);
```

Comment:
Here you should better return CreatedAtAction and specify the getBlogById route, plus route params, plus the object.